packages/lesswrong/server/manualMigrations/2023-06-13-renameShortformToQuicktakes.ts

```diff
@@ -0,0 +1,19 @@
+import { registerMigration } from "./migrationUtils";
+import { getSqlClientOrThrow } from '../../lib/sql/sqlClient';
+
+registerMigration({
+  name: "renameShortformToQuicktakes",
+  dateWritten: "2023-06-13",
+  idempotent: true,
+  action: async () => {
+    await getSqlClientOrThrow().none(`
+      UPDATE "Posts"
+      SET title = (
+        SELECT "displayName"
+        FROM "Users"
+        WHERE "Posts"."userId" = "Users"."_id"
+      ) || '''s Quick takes'
+      WHERE "shortform" IS TRUE
```

Comment:
Is there a reason this isn't an automatic migration?

Reply:
Only that it makes merge conflicts a lot less likely 😂